src/tests/init.rs

```diff
@@ -41,7 +41,12 @@ fn test_init_token() {
         pin_value.len() as CK_ULONG,
         std::ptr::null_mut(),
     );
-    assert_eq!(ret, CKR_PIN_INCORRECT);
+    if testtokn.dbtype == "nssdb" {
```

Comment:
I would skip the fn_init_otken entirely if no nssdb, rather than matching just he result ?

Reply:
Fine by me. I though keeping the test around and checking return value of the nssdb will make sure we do not regress even in these corner cases, but skipping is probably ok. Changing.